test/features/show.hide.timeout.feature

```diff
@@ -20,3 +20,10 @@ Feature: Present / Not present
     Then I wait txtTimeout from values dictionary ms
     Then divTimeout from main page should not be present
     Then txtHeader from main page should be present
+
+  Scenario: Check that element has correct text
```

Comment:
Looks like that these checking should be moved to the separate file, because `show.hide.timeout.feature` should not check element text. Not sure that we should create separate feature file for each checking, but we can for ex. create some "common.feature" or "then.feature" and put these steps to it.

Reply:
Maybe `verifications.feature` is a good place to keep all steps with checks